framework/components/ecl-page-headers/basic/basic.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  title: 'Basic',
+  label: 'Basic',
+  handle: 'ecl-page-headers--basic',
```

Comment:
I can't open preview windows on basic and default variants, probably because of the handlers.
https://drive.google.com/file/d/0B8knVVW86wNgV3JfMm0xRlFmQU0/view